app/src/main/java/com/h/pixeldroid/api/PixelfedAPI.kt

```diff
@@ -9,6 +10,7 @@ import retrofit2.converter.gson.GsonConverterFactory
 import retrofit2.http.*
 import retrofit2.http.Field
 
+
```

Comment:
lol

Reply:
creating space for future construction :wink: